src/components/TopBanner.tsx

```diff
@@ -322,8 +317,12 @@ export default function TopBannerCarousel(props: TopBannerCarouselProps) {
 
   const { deviceType, isMobile } = useDeviceType();
   // 터치 핸들링
-  const wrapperRef = React.useRef<HTMLDivElement>();
-  const touchRef = React.useRef(null);
+  const wrapperRef = React.useRef<HTMLDivElement>(null);
+  const touchRef: MutableRefObject<any> = React.useRef<{
+    id: any;
+    startX: any;
+    at: number;
+  }>(null);
```

Comment:
```suggestion
  const touchRef = React.useRef<{
    id: Touch['identifier'];
    startX: number;
    at: number;
  }>();
```

그리고 387번 줄을 `touchRef.current = undefined;`로 고쳐주세요